### ISSUE TITLE
Access to flowTracker cache

### ISSUE THREAD

**Issue by dandare100** [mushorg/go-dpi#72]
Thank you for supporting a really useful module/library.

I would like to check if there is a specific reason that access to the cache (var flowTracker *cache.Cache) is not possible from a client module.

I would like to reference the flow cache from my module and it looks easy enough to do(will require a small change to go-dpi on my side) but I don't want to get caught out with a valid reason not to do it.

The intent is read-only and it will be to "list" a current set of known and identified flows.

**Comment by glaslos** [comment]
Hi @dandare100 did you manage to solve your problem?
I generally don't see an issue exposing the cache 🤔

**Comment by dandare100** [comment]
Hello @glaslos 

I did come right thanks. Apologies, I wasn't sure on the activity regarding this project so I closed the issue.

I managed to get go-dpi integrated to the latest version of nDPI. From 3.2-stable to 4.4-stable
There were some method signature changes and the location of the header include files too.

I also noticed that in go-dpi every packet gets added to a "Flow" in GetFlowForPacket. This causes the system to use a large amount of memory (in the example) for big downloads (iso's etc), even for already identified Flows, so I changed the logic to skip the packet addition once the flow has been identified. This met my needs. I am not sure if there are any downstreams of this project that rely on all the packets being present in a Flow struct.

If this project is still in use I am happy to share the changes via pull requests

Regards

Jeff
